src/strings/en-gb.json

```diff
@@ -995,6 +995,8 @@
     "LabelBirthYear": "Birth year:",
     "LabelBirthDate": "Birth date:",
     "LabelBindToLocalNetworkAddress": "Bind to local network address:",
+    "LabelAutoCollection": "Auto create collection",
+    "LabelAutoCollectionHelp": "When at least 2 movies in the library have the same collection name, the collection will be automatically created",
```

Comment:
iirc, there is no need to add these lines to `en-GB` if they are the same as in `en-US`, because the latter is fallback language.

Reply:
OK. I removed changes in en-gb.json.